packages/api/src/lib/mmt.ts

```diff
@@ -410,3 +410,48 @@ export async function getSectionDependencies(mmtUrl: string, archive: string, fi
   }
   return dependencies;
 }
+export const getSparqlQueryForLoRelation = (uri: string, relationType: AllLoRelationTypes) => {
+  if (!uri) {
+    console.error('URI is absent');
+    return;
+  }
+  const isDimConceptType = ALL_DIM_CONCEPT_PAIR.includes(
+    relationType as LoRelationToDimAndConceptPair
+  );
+  const isOnlyConceptType = ALL_ONLY_CONCEPT.includes(relationType as LoRelationToConcept);
+
+  let query = '';
+  if (isDimConceptType) {
+    query = `
+        PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
```

Comment:
TODO: Use single query for all type1. And single query for all type2 relations.

Reply:
Done.

Reply:
Okay !